test-harness/modules/mocker/common.py

```diff
@@ -53,6 +53,13 @@ def _create_common_endpoints():
         res.append(ReflectingTcpIpEndpoint(ip='127.0.0.1', port=8181))
         # mesos
         res.append(MesosEndpoint(ip='127.0.0.2', port=5050))
+        # additional mesos instances for leader DNS changes tests
+        res.append(MesosEndpoint(ip='127.0.0.3', port=5050))
+        # mesos non-leading master that runs Mesos and AR
+        res.append(ReflectingTcpIpEndpoint(ip='127.0.0.10', port=5050))
+        res.append(ReflectingTcpIpEndpoint(ip='127.0.0.10', port=80))
```

Comment:
No need for these, we already have `ip='127.0.0.2', port=80` and `ip='127.0.0.3', port=80` a couple of lines below. Every time you add a new Endpoint it makes tests start&stop a bit slower so let's keep this number at minimum.